src/traces/contour/plot.js

```diff
@@ -64,15 +64,16 @@ function plotOne(gd, plotinfo, cd) {
         pathinfo = emptyPathinfo(contours, plotinfo, cd[0]);
 
     if(trace.visible !== true) {
```

Comment:
as above, we need to check again `true`, not truthy values as `visible` can be set to `'legendonly'`.